modules/local/report.nf

```diff
@@ -130,14 +130,13 @@ def generate_coverage_data(sample_data, bp_field, species){
                 && species[species_data_pos].fixed_genome_size != null){
 
                 def length = species[species_data_pos].fixed_genome_size.toLong()
-                def cov = base_pairs / q_length
+                def cov = base_pairs / length
                 entry.value[params.coverage_calc_fields.fixed_cov] = cov.round(2)
-
             }
 
         }
     }
-
+    return sample_data
```

Comment:
What was this returning before that you needed to add this return statement. Is this just for clarity (that is, by default it was already returning sample data)?

Reply:
Previously nothing was returned and the data was modified in place. While I was debugging the code I decided to explicitly return a value for the sake of testing. But I found that adding a return statement also made the code more readable/testable.